source/config/Config.h

```diff
@@ -333,6 +333,7 @@ namespace Aws
                 static constexpr char DEFAULT_SAMPLE_SHADOW_OUTPUT_DIR[] = "~/.aws-iot-device-client/sample-shadow/";
 
                 static constexpr char CLI_HELP[] = "--help";
+                static constexpr char CLI_VERSION[] = "--version";
```

Comment:
@RogerZhongAWS Can you add a short blurb to the Version doc in the repo that mentions customers can retrieve the version using the `--version` argument to the cli and include a short snippet.

https://github.com/awslabs/aws-iot-device-client/blob/main/docs/VERSION.md

Reply:
yes